src/nni_manager/training_service/pai/paiK8S/paiK8STrainingService.ts

```diff
@@ -78,18 +79,23 @@ class PAIK8STrainingService extends PAITrainingService {
                 }
                 break;
 
-            case TrialConfigMetadataKey.TRIAL_CONFIG:
+            case TrialConfigMetadataKey.TRIAL_CONFIG: {
```

Comment:
why need a bracket here?

Reply:
To fix eslint error `Unexpected lexical declaration in case block  no-case-declarations`.